app/screens/entry/entry.js

```diff
@@ -62,7 +62,7 @@ export default class Entry extends PureComponent {
         isLandscape: PropTypes.bool,
         enableTimezone: PropTypes.bool,
         deviceTimezone: PropTypes.string,
-        initializeModules: PropTypes.func.isRequired,
+        initializeModules: PropTypes.func,
```

Comment:
Not related, only I'm warning of this.  It's safe to remove isRequired since we're doing conditional check before calling this function.